pkg/config/config.go

```diff
@@ -99,12 +99,15 @@ const (
 	AuthCacheTTLKey     = "auth.cache.ttl"
 	AuthCacheJitterKey  = "auth.cache.jitter"
 
-	BlockstoreTypeKey                    = "blockstore.type"
-	BlockstoreLocalPathKey               = "blockstore.local.path"
-	BlockstoreS3RegionKey                = "blockstore.s3.region"
-	BlockstoreS3StreamingChunkSizeKey    = "blockstore.s3.streaming_chunk_size"
-	BlockstoreS3StreamingChunkTimeoutKey = "blockstore.s3.streaming_chunk_timeout"
-	BlockstoreS3MaxRetriesKey            = "blockstore.s3.max_retries"
+	BlockstoreTypeKey                         = "blockstore.type"
+	BlockstoreLocalPathKey                    = "blockstore.local.path"
+	BlockstoreS3RegionKey                     = "blockstore.s3.region"
+	BlockstoreS3StreamingChunkSizeKey         = "blockstore.s3.streaming_chunk_size"
+	BlockstoreS3StreamingChunkTimeoutKey      = "blockstore.s3.streaming_chunk_timeout"
+	BlockstoreS3MaxRetriesKey                 = "blockstore.s3.max_retries"
+	BlockstoreS3CredentialsAccessKeyIdKey     = "blockstore.s3.credentials.access_key_id"
+	BlockstoreS3CredentialsSecretAccessKeyKey = "blockstore.s3.credentials.secret_access_key"
+	BlockstoreS3CredentialsSessionTokenKey    = "blockstore.s3.credentials.session_token"
```

Comment:
I'm in the middle of a mega-cleanup of config.go, #1729.  These string keys will soon be gone (all of them, entirely), so please avoid fixing them for now.  I promise I shall eradicate them.
Sorry for not being specific about this when I said I was "doing something something".